packages/orchestration/src/orchestration-client.ts

```diff
@@ -56,6 +60,34 @@ export class OrchestrationClient {
   }
 }
 
+/**
+ * @internal
+ * Type guard to check if the config is of type OrchestrationModuleConfig.
+ */
+function isOrchestrationModuleConfig(
+  config: OrchestrationModuleConfig | string
+): config is OrchestrationModuleConfig {
+  return typeof config === 'object' && 'templating' in config;
```

Comment:
[pp] typeof this.config === 'string' (a simpler typeguard for cheking if its json